### PR TITLE
Boilerplate Setup

### PR DESCRIPTION
**Pull request by benhutton**

### CODE REVIEW

**Comment by benhutton** [comment]
@joeosburn-desiringgod let me know if there are other things you want me to get set up here

**Comment by benhutton** [comment]
@joeosburn-desiringgod i don't see it

File: components-rails.gemspec [hunk 1]
```diff
@@ -6,12 +6,11 @@ require "components/rails/version"
...
   spec.version       = Components::Rails::VERSION
-  spec.authors       = ["Ben Hutton"]
-  spec.email         = ["benhutton@gmail.com"]
+  spec.authors       = ["Joe Osburn", "Ben Hutton", "Desiring God"]
+  spec.email         = ["joe@jnodev.com", "benhutton@gmail.com", "web@desiringgod.org"]
 
-  spec.summary       = %q{TODO: Write a short summary, because Rubygems requires one.}
-  spec.description   = %q{TODO: Write a longer description or delete this line.}
-  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
+  spec.summary       = %q{Add Component support to Rails.}
```
Comment (joeosburn-desiringgod) [inline components-rails.gemspec]: @benhutton Hmm, sorry, my comment didn't show up. Might have been related to the outage, or possibly user error (on my part.)

Could we change this to say "web components" or possibly "front end components" or something like that?

Comment (benhutton) [inline components-rails.gemspec]: i'll go with "front end components". "web component" is more specific than we're trying to be

**Comment by benhutton** [comment]
@joeosburn-desiringgod all fixed

**Pull request merged**